target/linux/ipq40xx/image/Makefile

```diff
@@ -241,6 +241,21 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_emd1
+	$(call Device/FitImage)
+	DEVICE_VENDOR := EnGenius
+	DEVICE_MODEL := EMD1
+	DEVICE_DTS_CONFIG := config@4
+	BOARD_NAME := emd1
+	SOC := qcom-ipq4018
+	KERNEL_SIZE := 4096k
```

Comment:
Is this a hard limit? Fro my experience, >4MB large kernels can be loaded from SPI flash without a problem.

As it's not used anywhere, you should consider dropping the kernel size definition.

Reply:
It's not used here. I've removed it.